cmd/kubent/main_test.go

```diff
@@ -128,6 +127,7 @@ func TestMainExitCodes(t *testing.T) {
 		{"version long flag set", []string{"--version"}, 0, "", "", false},
 		{"empty text output", []string{clusterFlagDisabled, helm3FlagDisabled}, 0, "", "", false},
 		{"empty json output", []string{"-o=json", clusterFlagDisabled, helm3FlagDisabled}, 0, "[]\n", "", false},
+		{"fail to initialize collectors", []string{"-o=json", "-f=fail"}, 200, "[]\n", "", false},
```

Comment:
The test is probably OK, but the name is a bit misleading. The error, in this scenario with `-f=fail` flag does not come from collector init, but from the `Get` of the given collector, so maybe just drop the `initialize` bit.